### PR TITLE
Use brew bundle

### PR DESCRIPTION
**Pull request by stephengroat**
make sure your code would work on both windows, linux and osx

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1359)

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: d363e22ceb1441d2e641c3c41b06ed587c72d901
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: .travis.yml [hunk 1]
```diff
@@ -75,10 +75,7 @@ install: |-
     osx)
       MACOSX_DEPLOYMENT_TARGET=$DIST
       brew update # see travis-ci/travis-ci#8552
-      for p in gettext libtool automake zlib;
-        do brew list $p >/dev/null || brew install --verbose $p; done
-      brew link --force gettext libtool zlib
-      brew install --verbose wxmac --with-static --with-stl
+      brew bundle
```
Comment (slodki) [inline .travis.yml]: Thanks.
I'll add `--file` to move spec to subdir and `--verbose` to prevent travis timeout.

Comment (stephengroat) [reply .travis.yml]: sure. which subdir do you want?

Comment (slodki) [reply .travis.yml]: /util/

Comment (stephengroat) [reply .travis.yml]: done

**Comment by slodki** [comment]
`--verbose` is not working with `brew bundle` as expected - all wxmac compile is silent. See [log](https://travis-ci.org/moneymanagerex/moneymanagerex/jobs/291231166#L961)

**Comment by slodki** [comment]
We should compile wxmac from sources not using brew. This will produce same result but allow use ccache to speed up compile process. Using brew to compile static version of wxmac gives not benefit to us.

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 8c44191991ef437e31550daeeec99351651c752e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 8c44191991ef437e31550daeeec99351651c752e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 38cb0b80a370e7a3a567ec5578bae419998054c2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 8c44191991ef437e31550daeeec99351651c752e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 38cb0b80a370e7a3a567ec5578bae419998054c2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: a9954d4bd58b1d8e70791f29fac3d541fc8699d1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by stephengroat** [comment]
@slodki look at the changes that I've made. i've used `travis_wait` and the travis `directories` cache to cache the result of the `wxmac` brew output. does that look good?

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 8c44191991ef437e31550daeeec99351651c752e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 38cb0b80a370e7a3a567ec5578bae419998054c2
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: a9954d4bd58b1d8e70791f29fac3d541fc8699d1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 8de6a68a40553ab901b29d35bef26ffabedc43b6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by stephengroat** [comment]
@slodki updates have gotten the `brew bundle` to work and cache `brew` binaries and relink them from the cache.

let me know what you think

**Comment by slodki** [comment]
Now it fails with [`Linking /usr/local/Cellar/wxmac/3.0.3.1_1... 0 symlinks created`](https://travis-ci.org/moneymanagerex/moneymanagerex/jobs/292196831#L957). Could you diagnose and fix this?

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: ec2d8814c44522cc78d24ae90957af167b32f463
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by stephengroat** [comment]
@slodki should the cache between the two `xcode9.1` jobs be able to be re-used? is there any real difference between `DIST=10.12` and `DIST=10.13` at the dependency level?

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: ec2d8814c44522cc78d24ae90957af167b32f463
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: d2ecb30acea36ea516e10231f3350ed8c4ed0a50
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: ec2d8814c44522cc78d24ae90957af167b32f463
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: d2ecb30acea36ea516e10231f3350ed8c4ed0a50
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: fd430d604d1d6fd7383671fc07b338db3f707bc6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by slodki** [comment]
Each travis job has separate cache file not shared between jobs. But when cache is empty (not created before) for specific branch, then more general cache file is searched (form different branch etc.):
```
Setting up build cache
$ export CASHER_DIR=$HOME/.casher
$ Installing caching utilities
attempting to download cache archive
fetching testbranch/cache-linux-trusty-41df537ab8c8862c2f71e539987b8475313a6b8754b46966de81bf9a4572aad2--compiler-gcc.tgz
fetching testbranch/cache--compiler-gcc.tgz
fetching master/cache-linux-trusty-41df537ab8c8862c2f71e539987b8475313a6b8754b46966de81bf9a4572aad2--compiler-gcc.tgz
fetching master/cache--compiler-gcc.tgz
could not download cache
```

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: ec2d8814c44522cc78d24ae90957af167b32f463
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: d2ecb30acea36ea516e10231f3350ed8c4ed0a50
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: fd430d604d1d6fd7383671fc07b338db3f707bc6
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 0f6a3c935c37c82cae8167fa336d24ffd41497fa
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by slodki** [comment]
This PR breaks Mac builds - see Travis logs.

**Pull request merged**